server/protocols/webadmin/webadmin-jmap/src/test/java/org/apache/james/webadmin/data/jmap/UserIdentitiesRoutesTest.java

```diff
@@ -290,4 +292,219 @@ void listIdentitiesShouldReturnNotFoundWhenCanNotQueryDefaultIdentity() {
                 "    \"details\": null" +
                 "}");
     }
+
+    @Test
+    void createIdentityShouldWork() {
+        String creationRequest = "" +
+            "    {" +
+            "        \"name\": \"create name 1\"," +
+            "        \"email\": \"bob@domain.tld\"," +
+            "        \"mayDelete\": true," +
```

Comment:
that would go with my previous point.. you do declare the mayDelete field in your creation payload?

Reply:
My mistake (I copied the payload)
If `mayDelete` is always TRUE (server set), the `mayDelete` property in the payload is dangling

Reply:
`mayDelete` is false -> server-set
`mayDelete` is true -> user-set

instead?
Here I think we only be able to create/update user-set identity

Reply:
> mayDelete is false -> server-set
> mayDelete is true -> user-sett

this is what we want or this is currently already?

// Currently, the Jmap method (for user) is also can not set `mayDelete`

Reply:
Actually good point, in the spec it is supposed to be server-set only indeed.
Just remove this in the test, and maybe make sure with an other test that if a request comes with mayDelete to false, we should reject it, as for server-set values, in the spec, it says the client MUST NOT send this property when creating a new object of this type